pkg/service/main.go

```diff
@@ -39,32 +38,12 @@ func Main(serviceName string, serviceVersion string, proto interface{}, ctx cont
 	sdkFlags.Parse(os.Args[1:])
 
 	serviceName = setServiceName(serviceName, sdkFlags.Profile())
-	if serviceName == "" {
-		_, _ = fmt.Fprintf(os.Stderr, "Please specify device service name")
-		os.Exit(1)
-	}
-	common.ServiceName = serviceName
-	if serviceVersion == "" {
-		_, _ = fmt.Fprintf(os.Stderr, "Please specify device service version")
-		os.Exit(1)
-	}
-	common.ServiceVersion = serviceVersion
-	if driver, ok := proto.(dsModels.ProtocolDriver); ok {
-		common.Driver = driver
-	} else {
-		_, _ = fmt.Fprintf(os.Stderr, "Please implement and specify the protocoldriver")
-		os.Exit(1)
-	}
-	if discovery, ok := proto.(dsModels.ProtocolDiscovery); ok {
-		common.Discovery = discovery
-	} else {
-		common.Discovery = nil
-	}
+	sdk = &DeviceService{}
```

Comment:
Shouldn't this also be reverted to "service" or "svc" or "ds"?

Reply:
nice catch, gotta change it to `ds`